package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "wonder-stuff",
+  "version": "1.0.0",
+  "description": "Packages for sharing features across JavaScript-based projects",
+  "main": "index.js",
```

Comment:
Does it make sense to specify `main` if we're going to be publishing multiple packages?

Reply:
Just copied wonder blocks